### PR TITLE
feat: add stdout repository activity

### PR DESCRIPTION
**Pull request by mabdh**
- Add option to use logger instead of sinking to DB for activity logs

### CODE REVIEW

File: pkg/logger/logger.go [hunk 1]
```diff
@@ -8,9 +8,9 @@ import (
 	"go.uber.org/zap/zapcore"
 )
 
-func InitLogger(cfg Config) *log.Zap {
+func InitLogger(level, format string) *log.Zap {
```
Comment (ishanarya0) [inline pkg/logger/logger.go]: Where are we using `format` ?

**Review by ishanarya0 — CHANGES_REQUESTED** [review]

File: pkg/logger/logger.go (hunk 1 above)
Comment (mabdh) [reply pkg/logger/logger.go]: we are removing it, just updated it, can you please check again?

**Review by ishanarya0 — APPROVED** [review]
LGTM
let's also update the docs adding details of this configuration to chose from db and stdout

File: pkg/logger/logger.go (hunk 1 above)
Comment (ishanarya0) [reply pkg/logger/logger.go]: yes thanks

**Comment by mabdh** [comment]
> LGTM let's also update the docs adding details of this configuration to chose from db and stdout

sure things, will update the docs

**Pull request merged**